### PR TITLE
added an example to the doc

### PR DESCRIPTION
**Pull request by purnimagupta**
Wrote an example for creating a 1x3 Gluster volume. Also added screenshots but some of the pictures aren't rendering on github because I've used markdown for Sphinx. @sac @sankarshan

### CODE REVIEW

**Comment by nandajavarma** [comment]
@pooyaX14 Can you please rebase your branch with gdeploy master and do a force push? Looks like your branch is conflicting with the current master.

**Comment by purnimagupta** [comment]
Hi @nandajavarma , I have resolved it. Please have a look.

File: docs/source/examples.rst [hunk 1]
```diff
@@ -5,3 +5,77 @@ examples
...
+You can also check whether the files have been created and replicated thrice inside the directory ``/mnt/data1/1`` on the remote nodes by running the command::
+
+   $ ls
+
+.. image:: images/1x3_vol.png
+
+
+
+We can see that the files have been successfully replicated on all the three nodes.
+>>>>>>> b8f33e06e862706917c901ee66dad043e03953aa
```
Comment (nandajavarma) [inline docs/source/examples.rst]: Commit hash. Looks like you missed this when resolving the conflict. :-)

**Comment by purnimagupta** [comment]
@nandajavarma , removed it :)

**Pull request merged**